src/core/server/plugins/types.ts

```diff
@@ -278,6 +278,7 @@ export interface PluginInitializerContext<ConfigSchema = unknown> {
   env: {
     mode: EnvironmentMode;
     packageInfo: Readonly<PackageInfo>;
+    instanceUuid: string;
```

Comment:
`instanceUuid` is now accessible via `pluginInitializerContext.env.instanceUuid`, and the `Environment` service is no longer public.